### ISSUE TITLE
Allow array of regexes (or strings) for must_include, must_exclude

### ISSUE THREAD

**Issue by bbugh** [mergeability/mergeable#573]
Hi! 👋 This is a great tool!

I've run into a snag, though. I have a list of about 30 exact filenames that need to be matched in a monorepo (configuration, build related things), and doing it all as one regex is very difficult for readability and writability.

https://github.com/mergeability/mergeable/blob/01fd21a8f81e303ee30d660452047601ba2adf3f/lib/validators/contents.js#L21-L30

Changing these to allow an array in `regex` would be really helpful. Adding `match` like `begins_with` and `ends_with` that does an exact match would be even better to avoid accidental matches or having to mark every filename with boundaries.

Thanks!

**Comment by shine2lay** [comment]
@bbugh Hey! 

re `match`: for `begins_with` and `ends_with`, we just convert them to regex to match anyways, I see your point about potentially troublesome regex.

Have you try `and` or `or` option? I think they could be useful for your use case. 
something like this, they can also be nested

```
- do: contents
  and:
    - must_include:
        regex: 'doc/.*'
        message: 'Custom message...'
    - must_include:
        regex: 'changelog.md'
        message: 'Custom message...'
  or:
    - must_include:
        regex: 'package-lock.json'
        message: 'Custom message...'
    - must_include:
        regex: 'yarn.lock'
        message: 'Custom message...'
```

**Comment by bbugh** [comment]
Yeah, that was what prompted me to write this. Written out with `or`, that one validation is 99 repetitive lines long. 😬 Having a shorter way to write it like the other matchers would be 🔥

**Comment by shine2lay** [comment]
@bbugh oh, I see. In that case, I don't think I'll get around to this anytime soon, if you want to create a PR for this, I'll do my best to get it merged asap.

I think you just need to change code in this file to support `must_include array` for all validators, you'll need to add 'array' to supported settings too https://github.com/mergeability/mergeable/blob/01fd21a8f81e303ee30d660452047601ba2adf3f/lib/validators/options_processor/options/must_include.js

**Comment by bbugh** [comment]
@shine2lay thanks for the quick reply, happy to PR. 

It appears that `MustInclude` already supports an array type, am I reading this correctly?

https://github.com/mergeability/mergeable/blob/01fd21a8f81e303ee30d660452047601ba2adf3f/lib/validators/options_processor/options/must_include.js#L31-L41

If so, it seems like all I need to do is adjust the `must_include` schemas to be `regex: ['string', 'array']`. Does that seem right?

**Comment by shine2lay** [comment]
implemented in #575
